### PR TITLE
Add unittest for describe LimitRange

### PR DESCRIPTION
**Pull request by zhangxiaoyu-zidif**
**What this PR does / why we need it**:
Add unittest for describe LimitRange

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by dims** [comment]
@k8s-bot ok to test

**Comment by zhangxiaoyu-zidif** [comment]
ping @pwittrock for approval. Thanks.

**Comment by zhangxiaoyu-zidif** [comment]
@sig-cli-maintainers for approve. Thanks. =)

**Comment by zhangxiaoyu-zidif** [comment]
@fabianofranz 
PTAL

**Comment by zhangxiaoyu-zidif** [comment]
@fabianofranz 
PTAL

**Comment by zhangxiaoyu-zidif** [comment]
ping @pwittrock

File: pkg/printers/internalversion/describe_test.go [hunk 1]
```diff
@@ -157,6 +157,66 @@ func TestDescribeConfigMap(t *testing.T) {
...
+			},
+		},
+	})
+	c := &describeClient{T: t, Namespace: "foo", Interface: fake}
+	d := LimitRangeDescriber{c}
+	out, err := d.Describe("foo", "mylr", printers.DescriberSettings{ShowEvents: true})
+	if err != nil {
+		t.Errorf("unexpected error: %v", err)
+	}
+	if !strings.Contains(out, "foo") || !strings.Contains(out, "mylr") || !strings.Contains(out, "Pod") || !strings.Contains(out, "cpu") || !strings.Contains(out, "5m") || !strings.Contains(out, "100m") || !strings.Contains(out, "memory") || !strings.Contains(out, "100Mi") || !strings.Contains(out, "10000Mi") || !strings.Contains(out, "10") || !strings.Contains(out, "Container") || !strings.Contains(out, "cpu") || !strings.Contains(out, "5m") || !strings.Contains(out, "100m") || !strings.Contains(out, "memory") || !strings.Contains(out, "100Mi") || !strings.Contains(out, "10000Mi") || !strings.Contains(out, "10") || !strings.Contains(out, "10m") || !strings.Contains(out, "50m") || !strings.Contains(out, "200Mi") || !strings.Contains(out, "500Mi") || !strings.Contains(out, "PersistentVolumeClaim") || !strings.Contains(out, "storage") || !strings.Contains(out, "5Gi") || !strings.Contains(out, "10Gi") {
```
Comment (fabianofranz) [inline pkg/printers/internalversion/describe_test.go]: Make this something like 

```
checks := []string{"foo", "mylr", "Pod", ...}
for _, check := range checks {
  if !strings.Contains(out, check) {
    t.Errorf("unexpected out: %v", out)
  }
}
```

Comment (zhangxiaoyu-zidif) [inline pkg/printers/internalversion/describe_test.go]: OK, I will fix it soon. :)

**Comment by zhangxiaoyu-zidif** [comment]
@fabianofranz
PTAL

**Comment by fabianofranz** [comment]
/lgtm
/approve no-issue 
thanks @zhangxiaoyu-zidif!

**Pull request merged**